src/components/Countries.js

```diff
@@ -0,0 +1,91 @@
+import React, { useEffect, useState } from 'react';
+import { NavLink } from 'react-router-dom';
+import { useSelector, useDispatch } from 'react-redux';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSearch } from '@fortawesome/free-solid-svg-icons';
+import { fetchCountries } from '../redux/countries/countriesSlice';
+
+const Countries = () => {
+  const dispatch = useDispatch();
+  const countries = useSelector((state) => state.countries.countries);
+  // const { id } = useParams();
+  // const reg = id ? id.toLowerCase() : 'africa';
```

Comment:
- [ ] [OPTIONAL] Please remove all commented code from your codebase because it's a bad practice to commit commented-out code. That's what version control is for. If you need that code back, just fetch it from the repo :smile: